### PR TITLE
Test rsa x931pad

### PR DESCRIPTION
**Pull request by slontis**
##### Checklist

- [ ] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

File: test/recipes/15-test_rsax931.t [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+             '-pkeyopt', 'pad-mode:x931',
+             '-sign'])),
+  "RSA Sign with x931 padding using unsupported digest should fail");
+
+ok(run(app(['openssl', 'pkeyutl', '-in', $infile, '-rawin', '-inkey', $inkey,
+            '-digest', 'SHA256',
+            '-pkeyopt', 'pad-mode:oaep',
+            '-sign',
+            '-out', 'sigoaep.txt'])),
+  "RSA Sign with oaep padding  using SHA256");
```
Comment (paulidale) [inline test/recipes/15-test_rsax931.t:56]: double space

**Review by paulidale — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Review by beldmit — APPROVED** [review]
LGTM if I get the intentions correctly.

We have low-level tests, right?

**Comment by slontis** [comment]
The are a few tests for x931 in acvp_test

**Comment by paulidale** [comment]
Merged.

**Pull request closed without merging**